packages/fxa-admin-panel/src/components/AccountSearch/ConnectedServices/index.tsx

```diff
@@ -0,0 +1,175 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+import dateFormat from 'dateformat';
+import { AttachedClient, Location } from 'fxa-admin-server/src/graphql';
+import React from 'react';
+import { DATE_FORMAT, ResultTableRow } from '../Account';
+
+type Nullable<T> = T | null;
+
+export const NUMBER_OF_SERVICES_TO_SHOW = 3;
+
+export const ConnectedServices = ({
+  services,
+}: {
+  services?: Nullable<AttachedClient[]>;
+}) => {
+  if (services && services.length > 0) {
+    if (services.length > NUMBER_OF_SERVICES_TO_SHOW) {
+      return (
+        <details>
+          <summary className="hover:cursor-pointer text-violet-900 font-semibold mb-4">
+            Toggle viewing {services.length} connected services
+          </summary>
+          {services.map((service) => (
+            <ConnectedService
+              key={`${service.name}-${service.createdTime}`}
```

Comment:
I simplified the `key` here because I don't see a collision happening for `{name}-{createdTime}` and we aren't modifying/rerendering this section anyway.